Extensions/3D/JsExtension.js

```diff
@@ -2315,77 +2323,83 @@ module.exports = {
           pixiResourcesLoader
         );
 
-        const properties = associatedObjectConfiguration.getProperties();
-        this._defaultWidth = parseFloat(properties.get('width').getValue());
-        this._defaultHeight = parseFloat(properties.get('height').getValue());
-        this._defaultDepth = parseFloat(properties.get('depth').getValue());
+        this._defaultWidth = 1;
```

Comment:
Yes: the instance is not reading anymore the object properties at creation.
Instead, it creates its stuff and then update according to what is in the object, usually in the `update()` method. Usually, it compares a stored value against the value in the object (to avoid needless work)